udata/settings.py

```diff
@@ -37,6 +37,8 @@ class Defaults(object):
     CACHE_KEY_PREFIX = 'udata-cache'
     CACHE_TYPE = 'redis'
 
+    SERVER_NAME = 'localhost:7000'
```

Comment:
If you use `inv serve`, even without `honcho`, you have a problem with the traces on the standard output.

Therefore, personally, I prefer to use `python manage.py serve`, not `inv serve`.

If you want to solve the problem in the two situations, without using Defaults, I think we should define an intermediate structure between Debug and Defaults.

Alternatively, we could make that `inv serve` (which is used in development only, as far as I know) uses `Debug` instead of `Defaults`.